cirrus/server/pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "cirrus"
 version = "0.1.0"
 description = ""
-authors = [""]
+authors = ["Someone Something <somewhere@example.com>"]
```

Comment:
Is this intentionally "Someone Something"? (just curious)

Reply:
Ah sorry poetry got upset that the author didn't match that pattern so I put something in. I'll remove that, didn't mean to commit it.